lib/transport_web/router.ex

```diff
@@ -33,6 +33,12 @@ defmodule TransportWeb.Router do
       get "/datasets/:slug/_add", UserController, :add_badge_dataset
     end
 
+    scope "/discussions" do
```

Comment:
What's the difference between ```:post_discussion``` and ```post_discussion_id```? Like between thread and comment?

Reply:
exactly, I'm trying to stick to datagouvfr grammar, I don't like it neither.

Reply:
OK